### PR TITLE
Added the inital database setup with models and the general app

### PR DESCRIPTION
**Pull request by preethamcoder**
I created models for the user and the reviews. We can always change the reviews model up, I temporarily set it to "video title," but we can figure that out after we work some more things out. Please review and let me know.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.env
```
Comment (TracyMichaels) [inline .gitignore]: This is already in here on line 15

Comment (preethamcoder) [reply .gitignore]: My apologies! I will fix that now.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    	def hash_pwd(self, passw):
+                self.password = generate_password_hash(passw)
+
+    	def check_password(self, passw):
+        	return check_password_hash(self.password, passw)
+
+class Reviews(db.Model):
+	id = db.Column(db.Integer, primary_key=True, nullable=False)
+	video_title = db.Column(db.String(50), nullable=False)
+	user_id = db.Column(db.Integer, nullable=False)
```
Comment (TracyMichaels) [inline app.py:35]: Might be a good Idea to add a foreign key constraint on this to the user.id

File: app.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	def __repr__(self):
+	        return f"<{self.id}:{self.first_name}>"
+
+    	def hash_pwd(self, passw):
+                self.password = generate_password_hash(passw)
+
+    	def check_password(self, passw):
+        	return check_password_hash(self.password, passw)
+
+class Reviews(db.Model):
```
Comment (TracyMichaels) [inline app.py:32]: Add a string column for the Video Id as well

File: app.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+app = Flask(__name__)
+database = os.getenv('database')
+app.config["SQLALCHEMY_DATABASE_URI"] = database
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key = b"Nothing in here"
+db = SQLAlchemy(app)
+
+
+class User(db.Model, UserMixin):
```
Comment (TracyMichaels) [inline app.py:18]: Add doc string, linter will throw a fit if it's not there

File: app.py (hunk 3 above)
Comment (TracyMichaels) [inline app.py:32]: doc string here too

File: app.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import os
```
Comment (TracyMichaels) [inline app.py:1]: doc string above this line as well

**Pull request closed without merging**